### DISCUSSION TITLE
Squishbox no sound through audio jacks

### DISCUSSION THREAD

**Discussion by Melissaeboell** [GeekFunkLabs/fluidpatcher#77] [General]
Hello, I bought the pre-assembled squishbox and installed everything on it. It boots up just like it seems like it should, and I've run tests to make sure the rasp pi is recognizing the midi input, and it is, however no sound comes out through the 1/4" audio jacks. I think the rasp pi may be sending signals to the original aux in the rasp pi instead of the 1/4" audio jack. Is there anything you can think of to help troubleshoot this?

**Comment by albedozero** [comment]
Sure - try the command
```
speaker-test -Dhw:sndrpihifiberry
```
If you get the message "No such device", then the sound card isn't installed. Re-run the install script `curl -L git.io/squisbhox` and make sure to choose the `1. SquishBox` option and respond `yes` to installing the soundcard and rebooting the computer.

If you enter the command above and hear static through the connected 1/4" outputs, look in the file _/home/pi/SquishBox/squishboxconf.yaml_ and make sure you see the line `  audio.alsa.device: hw:sndrpihifiberry` in the `fluidsettings` section. Let me know if this fixes your issue.
